public/app/features/alerting/unified/components/rule-editor/alert-rule-form/simplifiedRouting/SimplifiedRouting.tsx

```diff
@@ -0,0 +1,258 @@
+    const selectedContactPoint = contactPointsInAlert?.find((cp) => cp.alertManager === am.name);
+    return { alertManager: am, selectedContactPoint: selectedContactPoint?.selectedContactPoint };
+  });
+
+  // use reducer to keep this alertManagersWithSelectedContactPoints in the state
+  const [alertManagersWithCPState, dispatch] = useReducer(receiversReducer, alertManagersWithSelectedContactPoints);
+
+  function getContactPointsForForm(alertManagersWithCP: AMContactPoint[]) {
+    return alertManagersWithCP.map((am) => {
+      return { alertManager: am.alertManager.name, selectedContactPoint: am.selectedContactPoint };
+    });
+  }
+
+  // whenever we update the receiversState we have to update the form too
+  useEffect(() => {
+    const contactPointsForForm = getContactPointsForForm(alertManagersWithCPState);
+    setValue('contactPoints', contactPointsForForm, { shouldValidate: false });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [alertManagersWithCPState]);
+
+  //todo: decide what to do when there some alert managers not postable
+  //const shouldShowAM = alertManagerMetaDataPostable.length > 1;
+  const shouldShowAM = true;
+  const hrefToContactPoints = '/alerting/notifications';
+
+  return alertManagersWithCPState.map((alertManagerContactPoint, index) => {
+    return (
+      <div key={index}>
+        <Stack direction="column">
+          {shouldShowAM && (
+            <Stack direction="row" alignItems="center">
+              <div className={styles.firstAlertManagerLine}></div>
+              <div className={styles.alertManagerName}>
+                {' '}
+                Alert manager:
+                <img src={alertManagerContactPoint.alertManager.img} alt="" className={styles.img} />
+                {alertManagerContactPoint.alertManager.name}
+              </div>
+              <div className={styles.secondAlertManagerLine}></div>
+            </Stack>
+          )}
+          <Stack direction="row" gap={1} alignItems="center">
+            <ContactPointSelector
+              selectedReceiver={alertManagerContactPoint.selectedContactPoint}
+              dispatch={dispatch}
+              alertManager={alertManagerContactPoint.alertManager}
+            />
+            <div className={styles.contactPointsLinkRow}>
+              <Text color="secondary">To browse contact points and create new ones go to</Text>
+              <a
+                href={createUrl(hrefToContactPoints)}
+                target="__blank"
+                className={styles.link}
+                rel="noopener"
+                aria-label="View alert rule"
+              >
+                Contact points
+                <Icon name={'external-link-alt'} size="sm" />
+              </a>
+            </div>
+          </Stack>
+        </Stack>
+      </div>
+    );
+  });
+}
+
+export const useReceiversMetadataMapByName = (receivers: Receiver[]): Map<string, ReceiverPluginMetadata> => {
```

Comment:
there was the `useReceiversMetadata` function that returned the same Map,  but the keys were the `Receiver` and in this case I need to get by receiver's `name`. Maybe we could refactor the other method and add a parameter to be able to get by name?